README.md

```diff
@@ -35,6 +35,15 @@ FileBeat version to use.
 
     filebeat_version: 1.1.1
 
+On Debian based Systems you may use a URL to install a specific .deb
+To do so set 
+
+    filebeat_use_apt_repo: false (default true)
```

Comment:
The documentation shows the default values, so this should really just be set to the value of `true`. I'd adjust the documentation above to say something like this instead:

On Debian-based systems, you may use a URL to install a specific `.deb`. To do so, change the following value to `false`, then (optionally) adjust the value of `filebeat_deb_baseurl` (which has a default value set for you).